client/src/reducers/ontology.js

```diff
@@ -0,0 +1,26 @@
+// jshint esversion: 6
+const Ontology = (
+  state = {
+    enabled: false, // are ontology terms enabled?
+    terms: null, // an array of term names, eg, ['cell', 'lung cell', ...]
+    loading: true
+  },
+  action
+) => {
+  switch (action.type) {
+    case "configuration load complete":
+      const enabled =
+        action.config?.parameters?.annotations_cell_ontology_enabled;
```

Comment:
this may set `enabled` to null (rather than false) which is unintended.   Please replace with:
```
const enabled =
        action.config?.parameters?.annotations_cell_ontology_enabled ?? false;
```

(added the nullish coallescing op & false value at end)